src/CMakeLists.txt

```diff
@@ -9,13 +10,15 @@ idf_component_register(SRCS
                            "lumen/hardware/button/led_button.cpp"
                            "lumen/hardware/button/rgb_button.cpp"
                            "lumen/hardware/button/rocker_switch.cpp"
+                           "lumen/hardware/display.cpp"
                            "lumen/hardware/led_pin.cpp"
                            "lumen/net/mdns.cpp"
                            "lumen/net/wifi.cpp"
                            "lumen/web/server.cpp"
                            "main.cpp"
                        INCLUDE_DIRS "."
                        REQUIRES
```

Comment:
Do you need to include the Arduino libraries?

Reply:
No, it's included by `ESP32-HUB75-MatrixPanel-DMA`.